pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3010,7 +3010,7 @@ public CompletableFuture<Void> onPoliciesUpdate(@Nonnull Policies data) {
         }
 
         updateTopicPolicyByNamespacePolicy(data);
-
+        checkReplicatedSubscriptionControllerState();
```

Comment:
If there is no subscription that enabled `replicatedSubscription`, we should not start the scheduled snapshot task, right?

Reply:
https://github.com/apache/pulsar/blob/62afe0194e0636be2a0468fa7f47f271442ba0c8/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java#L3479-L3494